Dockerfile

```diff
@@ -12,15 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM k8s.gcr.io/build-image/debian-base:bullseye-v1.1.0
+FROM k8s.gcr.io/build-image/debian-base:bullseye-v1.2.0
 
 ARG ARCH
 ARG binary=./bin/${ARCH}/nfsplugin
 COPY ${binary} /nfsplugin
 
-RUN apt update && apt-mark unhold libcap2
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2
 RUN clean-install ca-certificates mount nfs-common netbase
```

Comment:
If you join this into a single RUN, then files that don't need to be in the image get removed before the result of the RUN command is collected:

```suggestion
RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates mount nfs-common netbase
```

That works because `clean-install` removes the apt cache:
```
# cat /usr/local/bin/clean-install
#!/bin/sh

# Copyright 2021 The Kubernetes Authors.
...
# A script encapsulating a common Dockerimage pattern for installing packages
# and then cleaning up the unnecessary install artifacts.
# e.g. clean-install iptables ebtables conntrack

set -o errexit

if [ $# = 0 ]; then
  echo >&2 "No packages specified"
  exit 1
fi

apt-get update
apt-get install -y --no-install-recommends $@
apt-get clean -y
rm -rf \
   /var/cache/debconf/* \
   /var/lib/apt/lists/* \
   /var/log/* \
   /tmp/* \
   /var/tmp/*
```

Reply:
ha, that's quite smart, you are the expert!